rpcs3/Emu/RSX/VK/VKDraw.cpp

```diff
@@ -244,6 +244,14 @@ void VKGSRender::load_texture_env()
 		return false;
 	};
 
+	auto get_border_color = [&](const rsx::Texture auto& tex)
+	{
+		const auto require_border_color_remap = m_device->get_custom_border_color_support().require_border_color_remap;
```

Comment:
There's no real reason to use a variable here since the member shares the same name